src/api/services/apiKeyService.ts

```diff
@@ -0,0 +1,41 @@
+import { Participant } from '../entities/Participant';
+import { CreatedApiKeyDTO } from './adminServiceHelpers';
+
+export type ApiKeySecretsDTO = {
+  plaintextKey: string;
+  secret: string;
+  name: string;
+};
+
+export const createdApiKeyToApiKeySecrets = (apiKey: CreatedApiKeyDTO): ApiKeySecretsDTO => {
+  // eslint-disable-next-line camelcase
+  return {
+    plaintextKey: apiKey.plaintext_key,
+    secret: apiKey.authorizable.secret,
+    name: apiKey.authorizable.name,
+  };
+};
+
+export const getApiRoles = async (participant: Participant) => {
+  const participantCurr = await Participant.query()
+    .findById(participant.id)
+    .withGraphFetched('apiRoles');
+
+  if (!participantCurr?.apiRoles) {
```

Comment:
nit: I think `return participantCurr?.apiRoles ?? [];` reads a bit nicer.

Reply:
or whatever we end up calling that variable

Reply:
Ah yeah that's nicer